app/models/devise_token_auth/concerns/user.rb

```diff
@@ -182,7 +182,7 @@ def create_new_auth_token(client_id=nil)
       self.tokens.delete(oldest_token.first)
     end
 
-    self.save!
+    self.save!(validate: false)
```

Comment:
Would this allow a modified object to pass validations? For example if you modify the object then call this method it would save the changes without validation?